payment/src/test/java/org/killbill/billing/payment/core/TestPaymentMethodProcessorWithDB.java

```diff
@@ -99,7 +99,6 @@ public void testDeletePaymentMethod() throws Exception {
         Assert.assertEquals(history2.getAccountRecordId(), paymentMethodModelDao.getAccountRecordId());
         Assert.assertEquals(history2.getTenantRecordId(), paymentMethodModelDao.getTenantRecordId());
         Assert.assertEquals(history2.getExternalKey(), paymentMethodModelDao.getExternalKey());
-        // Note: upon deletion, the recorded state is the same as before the delete
```

Comment:
See, we knew about it! ;-)

Reply:
If by 'we' you mean the common pool of knowledge that we both have, then 'yes' !